src/main/java/frc/robot/subsystems/shooter/ShooterIOReal.java

```diff
@@ -18,16 +18,23 @@ public class ShooterIOReal implements ShooterIO {
 
   private final StatusSignal<Double> pivotVelocity = pivotMotor.getVelocity();
   private final StatusSignal<Double> pivotVoltage = pivotMotor.getMotorVoltage();
-  private final StatusSignal<Double> pivotAmps = pivotMotor.getStatorCurrent();
+  private final StatusSignal<Double> pivotStatorCurrentAmps = pivotMotor.getStatorCurrent();
+  private final StatusSignal<Double> pivotSupplyCurrentAmps = pivotMotor.getSupplyCurrent();
   private final StatusSignal<Double> pivotTempC = pivotMotor.getDeviceTemp();
   private final StatusSignal<Double> pivotRotations = pivotMotor.getPosition();
```

Comment:
This is repeated enough you should really make it a class. A record with a custom constructor that takes the motor would really clean this up trivially.